src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in

```diff
@@ -731,9 +753,9 @@ class MiniBatchDocumentation:
             normalizing_const      -- DOUBLE PRECISON. Default 1.0. The
                                       normalizing constant to use for
                                       standardizing arrays in independent_varname.
-            dependent_offset       -- INTEGER. If specified, shifts all dependent
-                                      variable values by this number (should
-                                      only be used for numeric types).
+            one_hot_encode_int_dep_var -- BOOLEAN. Default FALSE. Flag to one-hot
```

Comment:
We don't support this optional param anywhere (sql or py). This should be removed from the doc.